Xamarin.Forms.Core/CornerRadius.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Diagnostics;
+
+namespace Xamarin.Forms
+{
+	[DebuggerDisplay ("TopLeft={TopLeft}, TopRight={TopRight}, BottomLeft={BottomLeft}, BottomRight={BottomRight}")]
+	[TypeConverter (typeof (CornerRadiusTypeConverter))]
+	public struct CornerRadius
+	{
+		public double TopLeft { get; private set; }
```

Comment:
the `private set;` isn't even required, as the property is only set from the `.ctor`

Reply:
Ok!